frontend/src/components/LogInForm.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Link, NavLink } from "react-router-dom";
+import "./style/LogInForm.css";
+
+class LogInForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "Username",
+      password: "Password"
+    };
+  }
+
+  render() {
+    return (
+      <div className="logIn">
+        <h1 className="form">Log In</h1>
+             <label className="label">Username</label>
+                  
+        <input
+          className="logInInput"
+          type="text"
+          value={this.state.username}
+          onChange={event => {
+            this.setState({ username: event.target.value });
+          }}
+        />
+             <label className="label">Password</label>
```

Comment:
Indentation fluke

Reply:
Halloween is over, but still (un)dead code made its way through...should be definitely dead now. 
For the indentation, "Prettier" keeps saving my indentation this way, so I can't correct it without being corrected...Computer 1- Human 0 :(